setup.py

```diff
@@ -21,7 +21,6 @@
 	'plotly',
 	'matplotlib',
 	'sklearn',
-	'keras-model-specs>=0.0.28',
-	'setuptools==39.1.0'
+	'keras-model-specs>=0.0.28'
```

Comment:
I think you haven't pulled from Albert's latest PR:

```suggestion
	'keras-model-specs>=0.0.30'
```